.github/renovate.json

```diff
@@ -1,11 +1,18 @@
 {
 	"extends": ["github>balena-io/renovate-config"],
-	"prConcurrentLimit": 1,
+	"prConcurrentLimit": 3,
+	"prHourlyLimit": 1,
+	"rebasewhen": "automerge",
 	"packageRules": [
 		{
-			"matchUpdateTypes": ["major", "minor", "patch", "digest"],
+			"groupName": "only-manually-bumped-packages",
+			"matchPackageNames": ["balena-hup-action-utils"],
+			"matchUpdateTypes": [
+				"major",
+				"minor"
+			],
 			"automerge": false,
-			"labels": ["renovate", "dependencies", "{{depType}}", "major"]
+			"enabled": false
```

Comment:
Do you really want renovate to never open a PR for this, or would you like it to open a PR for convenience but never rebase it and disable automerge?

Reply:
If it's a major or a minor, which has a dependency of the proxy being bumped & deployed first, I think it makes sense to avoid opening the PR completely. I expect that who-ever does such a minor or major, will anyway have to bump it everywhere as part of shipping their project.

Reply:
ack, as long as you're okay with creating more PRs by hand

Reply:
Like I have always done :)